templates/zerver/integrations.html

```diff
@@ -1179,6 +1185,31 @@
 
     </div>
 
+    <div id="pingdom" class="integration-instructions">
+
+      <p>Zulip supports Pingdom integration and can notify you of
```

Comment:
The English here could be a little cleaner, e.g. "Zulip supports integration with Pingdom".  Might as well fix the similar pattenr in the Phabricator integration just above it.